index.php

```diff
@@ -38,12 +38,42 @@
 		</div>
 		<?php endif; ?>
 
+
+
+
  		<div class="pull-right visible-desktop home-lab">
 			<a data-toggle="modal" href="#tutorial_video_modal"><img src="img/screencast_thumb.png"></a>
 		</div>
-		<h1><?php echo $homepage_title; ?> <small><?php echo $homepage_subtitle; ?></small></h1>
 
-		<p class="lead">You can use labrador to find and download processed data or request new datasets.
+<h1><img class="pull-left visible-desktop" src=<?php echo $logo; ?> style="margin: 0 20px 10px 0;">
+
+		<?php echo $homepage_title; ?> <small><?php echo $homepage_subtitle; ?></small></h1>
+
+		<p class="lead">
+
+                <?php 
+
+                if(isset($data_root)){
+
+                echo 'System Status: &nbsp;';
```

Comment:
Would be good to wrap all of this in a conditional in case it doesn't work. eg.:

```php
$data_root_base  = preg_split('/\//', $data_root);
$space_left_cmd  = shell_exec ("df -h | grep $data_root_base[1]"); 
$space_headers   = preg_split('/\s+/', $space_left_cmd);
$available       = str_replace("P|T|G", "", $space_headers[1]);
$usednumber      = str_replace("P|T|G", "", $space_headers[2]);
if(is_numeric($available)){
  echo 'System Status: &nbsp;';
  // [..etc..]
```